setup.py

```diff
@@ -9,13 +9,13 @@
     long_description = f.read()
 
 setup(
-    name="sqlalchemy-logger",
-    version="0.1.1",
+    name="SQLAlchemyLogging",
+    version="0.1.2",
```

Comment:
this is good. by the way the version uses semantic versioning, read this if you haven't already  https://semver.org/